### DISCUSSION TITLE
How to get stream stats?

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5426] [FAQs]
I want to get some stats related to my live stream like the number of people watching it, and the duration of the live stream.
Is there an easy way to list this information?

Thank you

**Comment by USAMAWIZARD** [comment]
Hi
you can get these statistics information by calling [this](https://antmedia.io/rest/#/BroadcastRestService/getBroadcastStatistics) [REST API](https://antmedia.io/rest/#/BroadcastRestService/getBroadcastStatistics).

![image](https://github.com/ant-media/Ant-Media-Server/assets/47350008/f19f7d1a-ef1e-4a31-9076-9d72efe887e1)


you can get the duration of the stream by calling [this API](https://antmedia.io/rest/#/BroadcastRestService/getBroadcast).
![image](https://github.com/ant-media/Ant-Media-Server/assets/47350008/9d1a9e28-6a25-437c-bf3a-365d020a3525)
